src/av2/datasets/tbv/scene_clusters.py

```diff
@@ -0,0 +1,356 @@
+# <Copyright 2022, Argo AI, LLC. Released under the MIT license.>
+
+"""Clustering of TbV vehicle logs, by spatial location (scene).
+
+Note:
+   - Each log within a cluster shares some significant visual overlap with other logs within its cluster.
+   - These are not before/after pairs. In some cases, all logs in a cluster may be "after" a change.
+   - Each cluster has at least one log in the val or test set.
+   - Logs within each cluster are provided in chronological order.
+"""
+
+
+ATX_CLUSTERS = {
```

Comment:
```suggestion
ATX_CLUSTERS: Final[Dict[int, Tuple[str, ...]]] = {
```

My suggestion is to make these map from int to tuples of strs since they're static data.

Reply:
Good call @benjaminrwilson. Just updated to make this change.

Reply:
Actually, looks like `mypy` interprets `1: ("mystring")` as `int: str` instead of `int: Tuple[str,...]`.
```
src/av2/datasets/tbv/scene_clusters.py:20: error: Dict entry 2 has incompatible type "int": "str"; expected "int": "Tuple[str, ...]"
src/av2/datasets/tbv/scene_clusters.py:21: error: Dict entry 3 has incompatible type "int": "str"; expected "int": "Tuple[str, ...]"
src/av2/datasets/tbv/scene_clusters.py:26: error: Dict entry 5 has incompatible type "int": "str"; expected "int": "Tuple[str, ...]"
```
I think I'll have to revert it.

Reply:
Wouldn't we need to add a comma for a tuple literal?

E.g., `1: ("mystring",)`

Reply:
I see, yeah. I think that's a bit uglier though. I would prefer we keep it as list.